recipes/addons.rb

```diff
@@ -62,3 +62,7 @@
       done
     EOH
 end
+
+if node['kube-hops']['kfserving_enabled'].eql?("true")
```

Comment:
Use `casecmp?` instead which ignores the case.

Reply:
Done